parcels/examples/tutorial_nemo_curvilinear.ipynb

```diff
@@ -33,7 +33,17 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We can create a `FieldSet` just like we do for normal grids. Note that we need to provide an extra filename for the `mesh_mask` file that holds information on the Curvilinear grid."
+    "We can create a `FieldSet` just like we do for normal grids.\n",
+    "Note that NEMO is discretised on a C-grid. U and V velocities are not located on the same grid (see https://www.nemo-ocean.eu/doc/node19.html).\n",
```

Comment:
Is 'grid' the right word here, is 'not located on the same grid'? Perhaps 'location'?